applications/tari_console_wallet/src/automation/command_parser.rs

```diff
@@ -302,4 +328,13 @@ fn test_parse_command() {
     } else {
         panic!("Parsed public key is not the same as provided.");
     }
+
+    let command_str = "export-utxos csv_file utxo_list.csv".to_string();
+    let parsed = parse_command(&command_str).unwrap();
+
+    if let ParsedArgument::CSVFileName(file) = parsed.args[0].clone() {
+        assert_eq!(file, "utxo_list.csv".to_string());
```

Comment:
Why is this here?
This is not test code

Reply:
This one is inside a test.

Reply:
This is not a unit test, right?

Reply:
You are correct, this is a unit test. 
But is this not behind a cfg(test) flag?

Reply:
It's not behind a feature flag, obviously an oversight by the original author (me). It's not really that big a deal, but easy enough to add 

```rust
#[cfg(test)]
mod test {
```

Reply:
Done